helm_helpers.go

```diff
@@ -325,8 +325,8 @@ func getChartVersion(r *release) (string, string) {
 		return r.Version, ""
 	}
 	cmd := command{
-		Cmd:         "bash",
-		Args:        []string{"-c", "helm search " + r.Chart + " --version " + strconv.Quote(r.Version)},
+		Cmd:         "helm",
+		Args:        []string{"search", r.Chart, "--version", r.Version},
```

Comment:
Here, `strconv.Quote` is removed. I am not too sure what the outcome of doing this is.

Reply:
Please read my description:

> This solves the "quoting" problem, that is there when using a single string with all arguments.